### PR TITLE
fix(*): Change l2geth from 0.5.29 to 0.5.31

### PR DESCRIPTION
**Pull request by qbzzt**

### CODE REVIEW

File: src/docs/useful-tools/networks.md [hunk 1]
```diff
@@ -178,7 +178,7 @@ These parameters are mostly useful to people responsible for running and adminis
 - [Configuration parameters](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/deploy-config/goerli.json)
 - [Latest releases of `op-geth`](https://github.com/ethereum-optimism/op-geth/releases)
 - [Latest releases of `op-node`](https://console.cloud.google.com/artifacts/docker/oplabs-tools-artifacts/us/images/op-node)
-- [Legacy geth](https://hub.docker.com/layers/ethereumoptimism/l2geth/0.5.29/images/sha256-e40ea183c[…]e981408ccf9c07191667554c48b620e5eabc61521c3b3?context=explore)
+- [Legacy geth](https://hub.docker.com/layers/ethereumoptimism/l2geth/0.5.31/images/sha256-e40ea183c[…]e981408ccf9c07191667554c48b620e5eabc61521c3b3?context=explore)
```
Comment (trianglesphere) [inline src/docs/useful-tools/networks.md:181]:
```suggestion
- [Legacy geth](https://hub.docker.com/layers/ethereumoptimism/l2geth/0.5.31/images/sha256-5577036dc36d167d11f5ac49b91cc0a3d835574928a9563783c2e70309e5eb28?context=explore)
```

**Review by Nicca42 — APPROVED** [review]

File: src/docs/useful-tools/networks.md [hunk 2]
```diff
@@ -93,7 +93,7 @@ Links to download data directories will be posted here as soon as they are avail
 - [Configuration parameters](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/deploy-config/mainnet.json).
 - [Latest release of `op-geth`](https://github.com/ethereum-optimism/op-geth/releases/tag/v1.101105.2)
 - [Latest release of `op-node`](https://github.com/ethereum-optimism/optimism/releases/tag/op-node%2Fv1.1.0)
-- [Legacy geth](https://hub.docker.com/layers/ethereumoptimism/l2geth/0.5.29/images/sha256-e40ea183c[…]e981408ccf9c07191667554c48b620e5eabc61521c3b3?context=explore).
+- [Legacy geth](https://hub.docker.com/layers/ethereumoptimism/l2geth/0.5.31/images/sha256-e40ea183c[…]e981408ccf9c07191667554c48b620e5eabc61521c3b3?context=explore).
```
Comment (trianglesphere) [inline src/docs/useful-tools/networks.md:96]:
```suggestion
- [Legacy geth](https://hub.docker.com/layers/ethereumoptimism/l2geth/0.5.31/images/sha256-5577036dc36d167d11f5ac49b91cc0a3d835574928a9563783c2e70309e5eb28?context=explore).
```

**Review by trianglesphere — APPROVED** [review]
links do work as is, but the sha is in the url which was weird.

File: src/docs/useful-tools/networks.md (hunk 2 above)
Comment (K-Ho) [reply src/docs/useful-tools/networks.md:96]: yeah https://hub.docker.com/layers/ethereumoptimism/l2geth/0.5.31/images/sha256-5577036dc36d167d11f5ac49b91cc0a3d835574928a9563783c2e70309e5eb28?context=explore

**Pull request merged**